src/pages/all-on-one/all-on-one.html

```diff
@@ -7,20 +7,98 @@
+        </div>
+      </ion-col>
+    </ion-row>
+
+    <ion-row class="main-content manoeuver-section flex-none" [ngClass]="{'content-open': maneuversButton.isEnabled}">
+      <ion-col col-3>
+        <div class="form-section">
+          <ion-row>
+            <ion-col class="manoeuver-header" col-9>
+              <span>Reverse / Right</span>
+            </ion-col>
+            <ion-col col-3>
+              <input type="checkbox" />
+            </ion-col>
+          </ion-row>
+          <all-on-one-form-sub-element-hold-no-modal section="manoeuverRRControlv1" text="control"></all-on-one-form-sub-element-hold-no-modal>
+          <all-on-one-form-sub-element-hold-no-modal section="manoeuverRRObservationv1" text="observation"></all-on-one-form-sub-element-hold-no-modal>
+        </div>
+      </ion-col>
+      <ion-col col-3>
+        <div class="form-section">
+          <ion-row>
+            <ion-col class="manoeuver-header" col-9>
+              <span>Reverse Park R</span>
+            </ion-col>
+            <ion-col col-3>
+              <input type="checkbox" />
+            </ion-col>
+          </ion-row>
+          <all-on-one-form-sub-element-hold-no-modal section="manoeuverPRControlv1" text="control"></all-on-one-form-sub-element-hold-no-modal>
+          <all-on-one-form-sub-element-hold-no-modal section="manoeuverPRObservationv1" text="observation"></all-on-one-form-sub-element-hold-no-modal>
+        </div>
+      </ion-col>
+      <ion-col col-3>
+        <div class="form-section">
+          <ion-row>
+            <ion-col class="manoeuver-header" col-9>
+              <span>Reverse Park C</span>
+            </ion-col>
+            <ion-col col-3>
+              <input type="checkbox" />
+            </ion-col>
+          </ion-row>
+          <all-on-one-form-sub-element-hold-no-modal section="manoeuverRPCControlv1" text="control"></all-on-one-form-sub-element-hold-no-modal>
+          <all-on-one-form-sub-element-hold-no-modal section="manoeuverRPCObservationv1" text="observation"></all-on-one-form-sub-element-hold-no-modal>
+        </div>
+      </ion-col>
+      <ion-col col-3>
+        <div class="form-section">
+          <ion-row>
+            <ion-col class="manoeuver-header" col-9>
+              <span>Forward Park</span>
+            </ion-col>
+            <ion-col col-3>
+              <input type="checkbox" />
+            </ion-col>
+          </ion-row>
+          <all-on-one-form-sub-element-hold-no-modal section="manoeuverFPControlv1" text="control"></all-on-one-form-sub-element-hold-no-modal>
+          <all-on-one-form-sub-element-hold-no-modal section="manoeuverFPObservationv1" text="observation"></all-on-one-form-sub-element-hold-no-modal>
         </div>
+      </ion-col>
+    </ion-row>
 
+    <ion-row class="main-content" [ngClass]="{'scroll': maneuversButton.isEnabled}">
```

Comment:
You might want maneuvers button to set a few booleans rather than checking button state directly, some of these checks are a bit cryptic.

Reply:
there are two states that I'm using
1. whenever manoeuvres button is enabled 
2. whenever manoeuvres button is disabled
depends on them I'm adding removing classes, I tried make all classes self-explanatory